docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -1785,3 +1785,102 @@ which a build variant's `depends_on` overrides the task definition's
 - Tasks listed under a build variant.
 - The build variant definition.
 - The task definition.
+
+#### Specific Activation Override Hierarchy
```

Comment:
I think we should now link to this section [here](https://github.com/evergreen-ci/evergreen/blob/4bff1af0fa1a9d7c63fe091528c760c9d85b4e19/docs/Project-Configuration/Project-Configuration-Files.md#L630) instead of linking to the buildvariant page.

Reply:
Linked the page, and added info about `activate` to the page.